packages/stylelint-config/CHANGELOG.md

```diff
@@ -1,7 +1,22 @@
-# Change Log
+# Changelog
```

Comment:
This changelog will be updated automatically by Lerna when we release a new version. The changelog is generated based on commits using conventional commits.

Reply:
OK, thank you, I didn't know that. It's been reverted.